### DISCUSSION TITLE
Sale / Discount

### DISCUSSION THREAD

**Discussion by afewyards** [saleor/saleor#11420] [Ideas]
It would be brilliant to have the following. For percentile based discounts we would like to have a automatic rounding of price to a certain cent

-- example
we set the discount to 30% on a couple products with different prices. We then also set the rounding cents to 99.

1. pair of socks
normal price: 23,99
discounted price with only percentage taken off. 16,793
rounded down to 15,99
2. t-shirt
normal price: 35,99
discounted price with only percentage taken off. 25,193
rounded down to 24.99

This would save a ton of work on creating sales.

**Comment by timuric** [comment]
@afewyards is there a case when you would like to round up?

**Reply by afewyards** [reply]
If the 99 cent is set as the wanted end result it will not round it up to 00...

**Reply by timuric** [reply]
What if rounding creates a more dramatic effect:
Price after discount: 1.98
Price after rounding: 0,99

Perhaps rounding up at certain threshold makes more sense.

**Reply by afewyards** [reply]
Yeah in my examples I round down due to if you set a discount to 30% and if you round up it is less than 30% which can lead to upset customers. Hench the rounding down

**Reply by afewyards** [reply]
And if this option is per sale we have our own control on when to apply it for edge cases
